documentation/docs/faq.md

```diff
@@ -60,12 +60,7 @@ Transactions from Hummingbot are normal transactions conducted on exchanges; the
 
 ### What data do you collect when I use Hummingbot?
```

Comment:
@Hashira-Lars can you add back this content?  What data do you collect when I use Hummingbot?